### ISSUE TITLE
Any plans adding support for llama2.cpp?

### ISSUE THREAD

**Issue by ivandir** [bsilverthorn/maccarone#18]
It would be better to run this locally

**Comment by bsilverthorn** [comment]
I agree that it would be interesting to try other models, especially local models. Happy to accept patches. I won't have time to implement this myself in the near future, though.
